internal/indexer/moniker_test.go

```diff
@@ -15,7 +15,10 @@ func TestEmitExportMoniker(t *testing.T) {
 	w := &capturingWriter{}
 
 	indexer := &Indexer{
-		moduleName:            "github.com/sourcegraph/lsif-go",
+		repositoryRemote:      "github.com/sourcegraph/lsif-go",
+		repositoryRoot:        "/users/efritz/dev/sourcegraph/lsif-go",
+		projectRoot:           "/users/efritz/dev/sourcegraph/lsif-go",
+		moduleName:            "https://github.com/sourcegraph/lsif-go",
```

Comment:
how come the module name becomes `https://...`?

Reply:
I forgot to update the test when we change the moniker schemes for v1.4.0.